academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,170 @@
+
+  ![Channel state](images/channelsstate.png)
+
+* Port allocation - [ICS 5](https://github.com/cosmos/ibc/tree/master/spec/core/ics-005-port-allocation)
+  An IBC module can bind to any number of ports. Each port must be identified by a unique `portID`. Since IBC is designed to be secure with mutually distrusted modules operating on the same ledger, binding a port returns a dynamic object capability. To take action on a particular port (for example, an open channel with its portID), a module must provide the dynamic object capability to the IBC handler. This requirement prevents a malicious module from opening channels with ports it does not own. Thus, IBC modules are responsible for claiming the capability that is returned on `BindPort`.
+  
+  ![Dataflow between clients](images/dataflow.png)
+
+* Vector commitments -[ICS 23](https://github.com/cosmos/ibc/tree/master/spec/core/ics-023-vector-commitments)
+  The proof format that all implementations must be able to produce and verify.
+* Host requirements - [ICS 24](https://github.com/cosmos/ibc/tree/master/spec/core/ics-024-host-requirements)
+  The paths that all IBC implementations must use to commit IBC messages.
+* Relayer - [ICS 18](https://github.com/cosmos/ibc/tree/master/spec/relayer/ics-018-relayer-algorithms)
+  Relayer algorithms are the "physical" connection layer of IBC — off-chain processes responsible for relaying data between two chains running the IBC protocol by scanning the state of each chain, constructing appropriate datagrams, and executing them on the opposite chain as is allowed by the protocol.
+
+While this background information is useful, IBC modules do not need to interact at all with these lower-level abstractions.
+
+### IBC/APP - application layer
+
+IBC also offers ICS definitions for applications:
+
+![Available ICS definitions for applications](images/app.png)
+
+* Fungible token transfer - [ICS 20](https://github.com/cosmos/ibc/tree/master/spec/app/ics-020-fungible-token-transfer)
+  This logic constitutes a "fungible token transfer bridge module" interfacing between the IBC routing module and an existing asset tracking module on the host state machine.
+* Interchain Accounts - [ICS 27](https://github.com/cosmos/ibc/tree/master/spec/app/ics-027-interchain-accounts)
+  ICS-27 Interchain Accounts outlines a cross-chain account management protocol built on IBC. ICS-27-enabled chains can programmatically create accounts on other ICS-27-enabled chains and control these accounts via IBC transactions, instead of having to sign with a private key. Interchain Accounts contain all of the capabilities of a normal account (i.e. stake, send, vote) but instead are managed by a separate chain via IBC in a way such that the owner account on the controller chain retains full control over any interchain account(s) it registers on host chain(s).
```

Comment:
"... instead are managed by a separate chain via IBC in a way such that the owner **account** on the controller chain retains full control over any interchain account(s) it registers on host chain(s)."
Would use the owner 'module' rather to clear up a common misunderstanding